SECURITY.md

```diff
@@ -0,0 +1,14 @@
+# Security Policy
+
+## Supported Versions
+
+Use this section to tell people about which versions of your project are
+currently being supported with security updates.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| > 2.5  | :white_check_mark: |
+
+## Reporting a Vulnerability
+
+Send an email to autolab-dev@andrew.cmu.edu to report a vulnerability. 
```

Comment:
As part of responsible disclosure, be sure to mention that security vulnerabilities should not be publicly disclosed for at least <x> days, to allow us time to address the issue. Of course, also stuff like don't actually test on production, or view/edit other users' data

Also should probably run this one through Dave